src/main/java/creativei/manager/impl/InquiryManagerImpl.java

```diff
@@ -55,7 +61,7 @@ public ResponseObject create(InquiryVo inquiryVo) {
             return ResponseObject.getResponse(ue.getMessage(), ExceptionType.DUPLICATE_VALUE.getCode());
         } catch (InvalidParamRequest ipr) {
             logger.error(ipr.getMessage(), ipr);
-            return ResponseObject.getResponse(ExceptionType.INVALID_METHOD_PARAM.getMessage(), ExceptionType.INVALID_METHOD_PARAM.getCode());
+            return ResponseObject.getResponse(ipr.getMessage(), ExceptionType.INVALID_METHOD_PARAM.getCode());
```

Comment:
Instead of sending ipr.getMessage in response, try to send a more generic user friendly response error back

Reply:
issue solved in d5326cd